mobile/src/defaultViewComponents/defaultViewTableCell.js

```diff
@@ -98,7 +98,7 @@ export default class DefaultViewTableCell extends Component {
   }
 
   renderCurrentLocation = location =>
-    location === 'lostfound' ? t('atLostFound') : t('withPerson')
+    location === 'lostfound' ? `${t("currentLocal")} ${t('atLostFound')}` : `${t("currentLocal")} ${t('withPerson')}`
```

Comment:
Because of right-to-left languages like Arabic and Hebrew, we shouldn't concatenate here, but rather in the translation itself.  It looks like we should leave the `currentLocal` translations as before, and pass the location to the `t` translate function.